runtime/vm_function_test.go

```diff
@@ -12,6 +12,53 @@ func TestFunction(t *testing.T) {
 	expect(t, `f1 := func() {}; f2 := func() { return f1(); }; f1(); out = f2();`, nil, objects.UndefinedValue)
 	expect(t, `f := func(x) { x; }; out = f(5);`, nil, objects.UndefinedValue)
 
+	expect(t, `f := func(...x) { return x; }; out = f(1,2,3);`, nil, &objects.Array{Value: []objects.Object{
+		&objects.Int{Value: 1},
+		&objects.Int{Value: 2},
+		&objects.Int{Value: 3},
+	}})
+
+	expect(t, `f := func(a, b, ...x) { return [a, b, x]; }; out = f(8,9,1,2,3);`, nil, &objects.Array{Value: []objects.Object{
```

Comment:
you can do 

```golang
expect(t, `f := func(...x) { return x; }; out = f(1,2,3);`, nil, ARR{1, 2, 3})
```

Reply:
ah - i figured there had to be something. Thanks! fixed